### ISSUE TITLE
Exception in info method

### ISSUE THREAD

**Issue by sefx** [Nikeshbajaj/Linear_Feedback_Shift_Register#3]
Looks like self.seq starts off as a numpy.int32 and not a list like info() expects.

Steps to reproduce:

```python
import pylfsr
l = pylfsr.LFSR(fpoly=[8,7,5,3])
l.next()
l.info()

8 bit LFSR with feedback polynomial  x^8 + x^7 + x^5 + x^3 + 1
Expected Period (if polynomial is primitive) =  255
Current :
 State        :  [0 1 1 1 1 1 1 1]
 Count        :  1
 Output bit   :  1
 feedback bit :  0
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-4-91f548e48c86> in <module>
----> 1 l.info()

c:\python37\lib\site-packages\pylfsr\pylfsr.py in info(self)
    247                 print(' feedback bit : ', self.feedbackbit)
    248                 if self.count > 0 and self.count < 1000:
--> 249                         print(' Output Sequence %s' % (''.join([str(int(x)) for x in self.seq])))
    250
    251         def check(self):
```

**Comment by Nikeshbajaj** [comment]
Thanks, Good catch, yes, .seq start with a bit, rather than an array. so .info method fails if there only one cycle is executed. For more than one, it works fine. 
Should be fixed in updated version @todo @Nikeshbajaj

**Comment by Nikeshbajaj** [comment]
Issue is fixed in [1.0.5](https://pypi.org/project/pylfsr/1.0.5/)
